src/ScottLogic.Internal.Training/ScottLogic.Internal.Training.Api/Controllers/ValuesController.cs

```diff
@@ -9,9 +9,9 @@ public class ValuesController : ControllerBase
     {
```

Comment:
Feel free to delete this class, it is only added as a example, it is no longer needed now that you have your own controllers.